script/openqa-webui

```diff
@@ -0,0 +1,2 @@
+# Our API commands are very expensive, so the default timeouts are too tight
+exec $(dirname $0)/openqa prefork -m production --proxy -i 100 -H 400 -w 30 -c 1 -G 800 "$@"
```

Comment:
This is now inconsistent with script/openqa-websockets and script/openqa-scheduler and script/openqa-livehandler. These scripts will drop all default arguments when one argument has been specified explicitly. This script on the other hand will keep the defaults. Not sure how well overriding arguments works that way.

Reply:
Yes, you are right. The problem I saw is that "script/openqa" is not only used for webui but also for gru and others. However I tried to solve that now in a similar way. So there is no `script/openqa-webui` anymore, just `script/openqa` but I added `script/openqa-gru` instead.